metrics/manager/manager_test.go

```diff
@@ -27,11 +27,11 @@ func TestFlow(t *testing.T) {
 	sink := util.NewDummySink("sink", time.Millisecond)
 	processor := util.NewDummyDataProcessor(time.Millisecond)
 
-	manager, _ := NewManager(source, []core.DataProcessor{processor}, sink, time.Second, time.Millisecond, 1)
+	manager, _ := NewManager(source, []core.DataProcessor{processor}, sink, MinMetricResolution, time.Millisecond, 1)
 	manager.Start()
 
 	// 4-5 cycles
-	time.Sleep(time.Millisecond * 4500)
+	time.Sleep(time.Millisecond * 4500 * 30)
```

Comment:
I'm not entirely certain what the intention here is of multiplying by 30.  Please, let's not have magic numbers (at least not without better comments).